### PR TITLE
[CO-145] Add build job for vestal_versions

### PR DESCRIPTION
**Pull request by vseoane**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# ENV BUNDLE_USER_HOME $HOME/.bundle
+# ENV BUNDLE_PATH $HOME/.bundle
+
+USER builder
+ENV HOME /home/builder
+RUN mkdir -p $HOME/app
+ADD --chown=builder:builder . $HOME/app/
+WORKDIR $HOME/app/
+
+RUN gem install bundler -v 2.0.1 && bundle install
```
Comment (careyrp) [inline Dockerfile]: Change to 1.15.4

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+            stages {
+                stage('Build Docker Image') {
+                    steps {
+                        sh """
+                        docker build -t ${dockerImage} .
+                        """
+                    }
+                }
+                /* // Testing is failing. Skip to unblock project to
```
Comment (careyrp) [inline Jenkinsfile]: Did you see if rake run test works?

File: Dockerfile (hunk 1 above)
Comment (careyrp) [reply Dockerfile]: Use the new docker file at https://github.com/dataxu/auth0-api/blob/0ed95a4ae0418178ea8bd5f56edc02be14562b95/Dockerfile

File: Jenkinsfile (hunk 2 above)
Comment (vseoane) [reply Jenkinsfile]: failed https://ci01.devaws.dataxu.net/job/vestal_versions/view/change-requests/job/PR-2/4/

**Review by careyrp — APPROVED** [review]

**Pull request merged**